x-pack/legacy/plugins/task_manager/task_manager.ts

```diff
@@ -85,7 +87,7 @@ export class TaskManager {
       index: opts.config.get('xpack.task_manager.index'),
       maxAttempts: opts.config.get('xpack.task_manager.max_attempts'),
       definitions: this.definitions,
-      taskManagerId: generateTaskManagerUUID(this.logger),
+      taskManagerId,
```

Comment:
Maybe prepend `kibana:` to the UUID, to indicate (to humans) where the value came from?

```js
const kibanaId = opts.config.get('server.uuid');

// ...

taskManagerId: `kibana:${kibanaId}`
```

Reply:
Yeah, no problem with that - now added.